common/ruletypes.h

```diff
@@ -445,7 +445,7 @@ RULE_BOOL(Spells, IllusionsAlwaysPersist, false, "Allows Illusions to persist be
 RULE_BOOL(Spells, UseItemCastMessage, false, "Enable to use the \"item begins to glow\" messages when casting from an item.")
 RULE_BOOL(Spells, TargetsTargetRequiresCombatRange, true, "Disable to remove combat range requirement from Target's Target Spell Target Type")
 RULE_BOOL(Spells, NPCBuffLevelRestrictions, false, "Impose BuffLevelRestrictions on NPCs if true")
-RULE_BOOL(Spells, ResurrectionEffectsBlock, true, "If enabled, resurrection effects cannot be overwritten.")
+RULE_INT(Spells, ResurrectionEffectBlock, 2, "0 = allow overwrites/rule disabled. If set to 1 = Block all buffs that would overwrite Resurrection Effects. If set to 2 = Will not overwrite Resurrection Effects, instead moves new buff to an empty slot if available. ")
```

Comment:
Changing the type of rule like this will not fix peoples’ database values. May want to put a required SQL in that sets their database values based on if they had this rule set to `true`/`false` prior to this change.

Reply:
Issue is it will set the new rule to this value and it won't be based on the old rule, as it will auto delete the old one, meaning this could override the settings an operator already had in place.

Reply:
Do you mind verifying if this is the correct way you'd like this done?